### DISCUSSION TITLE
Can React-Calendar be used to create range selection with hours?

### DISCUSSION THREAD

**Discussion by cubecleveland** [wojtekmaj/react-calendar#568] [Q&A]
Hello,
we are trying to achieve the following flow.

A calendar that will show a  weekly (7 day)  layout where all hour of the day are present.
Use can then range select inside the calendar the time and days he wants and ( for example picking 8Am to 12PM on a specific wedensday.
Is that possible with this ilbrary. if so is there any example we can look at?

**Comment by wojtekmaj** [comment]
It seems you'd require two things that are not available in React-Calendar:

 * Weekly calendar
 * Hour selection
 
So, the answer is no.
